README.md

```diff
@@ -24,6 +24,44 @@ Download a copy of Waterfall.jar from our buildserver here: [Waterfall](https://
 
 Waterfall requires **Java 8** or above.
 
+## How To (Plugin Developers)
+------
+ * See our API patches [here](BungeeCord-Patches)
+ * Waterfall API javadocs here: [papermc.io/javadocs](https://papermc.io/javadocs/)
```

Comment:
Should just lead to https://papermc.io/javadocs/waterfall, I understand Paper just leads to the javadocs page, but it has multiple options, and Waterfall just one.